### ISSUE TITLE
Certificate verification error installing devcontainer

### ISSUE THREAD

**Issue by ArtyMaury** [microsoft/vscode#77862]
When I try to start a devcontainer I get this error:
```
Installing VS Code Server for commit 2213894ea0415ee8c85c5eea0d0ff81ecc191529
Run: docker exec 7602314dc8df0bdf6cd43091a3643b3c89f5ff63ec1a463be93b1ddd3c3c192c mkdir -p /root/.vscode-server/bin/2213894ea0415ee8c85c5eea0d0ff81ecc191529_1563954742342
Error: unable to verify the first certificate
        at TLSSocket.onConnectSecure (_tls_wrap.js:1049:34)
        at TLSSocket.emit (events.js:182:13)
        at TLSSocket._finishInit (_tls_wrap.js:631:8)
```

I have to use a proxy that is defined in vscode settings. I've deactivated the 'http strict ssl' setting. I've tried to redefine those settings in the devcontainer settings but the certificates keep being verified.

- VSCode Version: 1.36.1
- OS Version: W10

**Comment by chrmarti** [comment]
Do you have the required certificate installed in the OS? If you do, could you check if this is still a problem with the latest Insiders build of VS Code? (Download: https://code.visualstudio.com/insiders/) We just fixed a related issue on Windows (#79044).

**Comment by ArtyMaury** [comment]
Hi,
Yes, the certificate was installed on the OS.
On the latest insiders version the problem is fixed!

Thanks a lot!
